x-pack/plugins/transform/public/app/sections/transform_management/components/action_schedule_now/schedule_now_action_name.tsx

```diff
@@ -31,11 +33,15 @@ export const isScheduleNowActionDisabled = (
   // Disable schedule-now for batch transforms which have completed.
   const completedBatchTransform = items.some((i: TransformListRow) => isCompletedBatchTransform(i));
 
+  // Disable start for transforms if stats does not exist
```

Comment:
Should be `Disable schedule now...`

Reply:
Removed here [`82de19d` (#166320)](https://github.com/elastic/kibana/pull/166320/commits/82de19d62bea414640929fd4b106c105a51e1a53)